lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -482,6 +492,10 @@ def administer(self, command, params, token):
                 f"You are not authorized. Please request a role from {self.admin_roles}"
             )
 
+    def check_job_permission(self, job: Job):
```

Comment:
Is this unused?

Reply:
Oops, yeah that's meant for the next PR. I'll remove it.

Reply:
Removed.